addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -147,21 +151,25 @@ public MaxCubeBridgeHandler(Bridge br) {
     private Socket socket = null;
     private BufferedReader reader = null;
     private OutputStreamWriter writer = null;
+    private final Lock socketLock = new ReentrantLock();
 
     private boolean previousOnline = false;
 
     private Set<DeviceStatusListener> deviceStatusListeners = new CopyOnWriteArraySet<>();
 
+    private final Lock pollingJobLock = new ReentrantLock();
```

Comment:
Starting the handler twice would be an significant issue of the framework. Handlers are only started once per thing.
We should avoid to create complexity to work around (unlikely) possible bugs of the framework.

MAX Binding should be able to run for months stable (and can...), so if in your case it runs only for 12h indeed we need to find the root cause and fix that.
There must be a combination of factors driving the instability you experience, let's find & fix that

The main purpose of the synchronized block is to avoid we are sending a command and send another one prior receiving the response of the first. 
Only in the more exceptional cases (e.g. during new device inclusion)  there is a significant delay between the sending of the command and the response. In all other cases, response is immediately, and the commands are consumed from the queue with a fixed delay, basically excluding the chance for clashes.